### PR TITLE
ReferenceFields: label hidden on the top of field

### PR DESCRIPTION
**Pull request by philippe-cleany**
closes: #827

### CODE REVIEW

File: src/mui/field/ReferenceField.js [hunk 1]
```diff
@@ -113,6 +110,14 @@ function mapStateToProps(state, props) {
...
 }
 
-export default connect(mapStateToProps, {
+const ConnectedReferenceField = connect(mapStateToProps, {
     crudGetManyAccumulate: crudGetManyAccumulateAction,
 })(ReferenceField);
+
+ConnectedReferenceField.defaultProps = {
+    addLabel: true,
+    referenceRecord: null,
```
Comment (fzaninotto) [inline src/mui/field/ReferenceField.js]: You don't need to move `referenceRecord` or `record` there.

File: docs/Fields.md [hunk 2]
```diff
@@ -26,7 +26,9 @@ export const PostList = (props) => (
 All field components accept the following attributes:
 
 * `source`: Property name of your entity to view/edit. This attribute is required.
-* `label`: Used as a table header of an input label. Defaults to the `source` when omitted.
+* `label`: In a datagrid, used as a table header of an input label.
```
Comment (fzaninotto) [inline docs/Fields.md]: please undo this change. You're talking about specifics in a place where the reader expects general information.

Comment (philippe-cleany) [inline docs/Fields.md]: Do you want me to also remove the part on addLabel ?

Comment (fzaninotto) [inline docs/Fields.md]: no, that part is fine.

**Comment by fzaninotto** [comment]
Thanks!
